### PR TITLE
Simple Inventory Projection

### PR DESCRIPTION
**Pull request by yu-peng**
Added simple LP model for calculating inventory projection and additional supplies. Given current inventory, projected demand, and known supply arrival schedule, calculate the inventory level of each day in the given date range and the amount of shortage of each day.

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -16,6 +16,7 @@ django-lockdown = "*"
 gunicorn = "*"
 dj-database-url = "*"
 whitenoise = "*"
+ortools = "*"
```
Comment (rcoh) [inline Pipfile:19]: this needs an accompanying change to `Pipefile.lock` -- I can rebuild it if you can't.

**Comment by mjacksonw** [comment]
One thing I recall Daniel saying on a call is that they're not going to want to ramp down supply even if all covid patients disappeared tomorrow – they figure they're going to need this supply for a while, so we should assume they'll want to be on the surplus side if we're using patient projections...

**Comment by yu-peng** [comment]
> Cool! Did you make sure the numbers are plausible?

Yes! Added in 
38487d1

File: Pipfile (hunk 1 above)
Comment (yu-peng) [reply Pipfile:19]: Thanks!

**Comment by yu-peng** [comment]
> can you move this into another file, or a CSV or something? We will move it into an upload / DB model.

Added in 
0943b0c

File: nyc_data/ppe/tests.py [hunk 2]
```diff
@@ -36,35 +36,63 @@ def setUp(self) -> None:
...
+            use_ihme_projection=False
         )
         self.assertEqual(
             future_rollup[dc.Item.gown], AssetRollup(asset=dc.Item.gown, demand=42, sell=1005)
         )
 
     def test_mayoral_rollup(self):
         rollup = aggregations.asset_rollup(
-            datetime.now() - timedelta(days=28), datetime.now(),
+            datetime.strptime('2020-04-12', '%Y-%m-%d') - timedelta(days=28), datetime.strptime('2020-04-12', '%Y-%m-%d'),
```
Comment (rcoh) [inline nyc_data/ppe/tests.py:72]: 1. Use the date constructor directly: `datetime.date(year=2019, month=4, day=13)`
2. Define this date once, name it "`today`" and use it everywhere :-)

File: public-data/ihme_projection_new_york.json [hunk 3]
```diff
@@ -0,0 +1,75 @@
+{
```
Comment (rcoh) [inline public-data/ihme_projection_new_york.json:1]: Can you add a README to this folder about where this data comes from and how it might updated?

Comment (yu-peng) [reply public-data/ihme_projection_new_york.json:1]: Fixed in 
6015065

File: nyc_data/ppe/tests.py (hunk 2 above)
Comment (yu-peng) [reply nyc_data/ppe/tests.py]: Fixed in 
a2f452c

**Pull request merged**